.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,10 +2,10 @@
 Fill in the relevant information below to help triage your issue.
 
 Pick the target branch based on the following criteria:
-  * Documentation improvement: master branch
```

Comment:
We don't use `master` nor `main` anymore: we should tell that we either request PRS to the default `X.Y.x` branch, or the oldest supported `X.Y.x`

Reply:
Can you check now, pls?